File: aiter_zadania/aiter/alg_iteracyjny.cpp
/*
 * alg_iteracyjny.cpp
 *
 */

#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
    int n;
    float iloczyn = 1;
    float a;
    cout << "Podaj liczbę całkowitą: ";
    cin >> n;
    for (int i=1; i <= n; i++) {
        cout << "Podaj liczbę: ";
        cin >> a;
        iloczyn *= a;
    }
    cout << iloczyn;
    return 0;
}

File: aiter_zadania/aiter/alg_iteracyjny.py
#! /usr/bin/env python
# -*- coding: utf-8 -*-

def main(args):
    n = int(input("Podaj liczbę całkowitą: "))
    iloczyn = 1
    for i in range(1, n+1):
        a = float(input("Podaj liczbę: "))
        iloczyn *= a
    print(iloczyn)

    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
